submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>papap</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <nav>
+        <ul class="header-icons">
+
+            <li class="greatest">
+                <label for="sq">
+                    <img class="h-image" src="icons/popup-button.png" alt="button">
+                </label>
+                <input tabindex="1" id="sq" type="checkbox" class="visually-hidden" />
```

Comment:
By default input and other interactive elements have focus in the browser, no need to add `tabindex` attribute if you don't want to change default order of interactive elements.